pyproject.toml

```diff
@@ -83,7 +77,7 @@ docs = [
     "qiskit-sphinx-theme>=1.14.0, <2"
 ]
 notebook-dependencies = [
-    "circuit-knitting-toolbox[cplex]",
+    "circuit-knitting-toolbox",
```

Comment:
This line doesn't hurt anything, but it doesn't do anything either since no other optional dependencies are referenced.
```suggestion
```